debezium-testing/debezium-testing-system/src/test/java/io/debezium/testing/system/tests/MongoConnectorTest.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.testing.system.tests;
+
+import io.debezium.testing.system.assertions.KafkaAssertions;
+import io.debezium.testing.system.tools.kafka.ConnectorConfigBuilder;
+import io.debezium.testing.system.tools.kafka.KafkaConnectController;
+import io.debezium.testing.system.tools.kafka.KafkaController;
+
+public class MongoConnectorTest extends ConnectorTest {
```

Comment:
I'm likely overlooking something. How is this different from `SqlConnectorTest` and why we need these 2 classes?